app/src/ui/diff/diff-helpers.tsx

```diff
@@ -387,3 +387,32 @@ export function getLargestLineNumber(hunks: DiffHunk[]): number {
 export function getNumberOfDigits(val: number): number {
   return (Math.log(val) * Math.LOG10E + 1) | 0
 }
+
+export function getFirstAndLastClasses(
```

Comment:
Woah, this function is infinitely better than the hack of nested if statements I set you up for, nicely done!